addons/event/tests/common.py

```diff
@@ -96,6 +96,11 @@ def setUpClass(cls):
             'date_tz': 'Europe/Brussels',
         })
 
+        cls.mail_template = cls.env['mail.template'].create({
```

Comment:
Does not seem necessary to have a variable defined in common setup for a single use ... we try to keep setup as low as possible :)